runtime/common/src/weights/pallet_contracts.rs

```diff
@@ -1004,17 +1070,19 @@ impl<T: frame_system::Config> pallet_contracts::weights::WeightInfo for WeightIn
     /// Proof: Contracts OwnerInfoOf (max_values: None, max_size: Some(88), added: 2563, mode: Ignored)
     /// Storage: System EventTopics (r:1602 w:1602)
     /// Proof Skipped: System EventTopics (max_values: None, max_size: None, mode: Ignored)
+    /// The range of component `r` is `[0, 20]`.
     fn seal_instantiate(r: u32) -> Weight {
         // Proof Size summary in bytes:
         //  Measured:  `1512 + r * (25573 ±0)`
         //  Estimated: `0`
-        // Minimum execution time: 384_252 nanoseconds.
-        Weight::from_ref_time(466_982_000)
-            // Standard Error: 66_870_986
-            .saturating_add(Weight::from_ref_time(48_895_356_612).saturating_mul(r.into()))
-            .saturating_add(T::DbWeight::get().reads(8_u64))
+        // Minimum execution time: 543_002 nanoseconds.
+        Weight::from_parts(626_922_000, 0)
+            // Standard Error: 81_700_194
+            .saturating_add(Weight::from_parts(61_961_443_354, 0).saturating_mul(r.into()))
+            .saturating_add(T::DbWeight::get().reads(8))
             .saturating_add(T::DbWeight::get().reads((400_u64).saturating_mul(r.into())))
-            .saturating_add(T::DbWeight::get().writes(5_u64))
+            .saturating_add(T::DbWeight::get().writes(5))
+            .saturating_add(T::DbWeight::get().writes((400_u64).saturating_mul(r.into())))
```

Comment:
What the hell! 400 writes times a variable r

Reply:
Guess it's fine for instantiating a smart contract, but still massive.